frontends/ol-components/src/components/LearningResourceExpanded/InfoSection.tsx

```diff
@@ -163,8 +189,8 @@ const INFO_ITEMS: InfoItemConfig = [
 
 type InfoItemProps = {
   label: string
-  Icon: RemixiconComponentType
-  value: string | number | null
+  Icon: RemixiconComponentType | null
+  value: string | number | null | ReactElement
```

Comment:
```suggestion
  value: React.ReactNode
```
which is a little broader but I believe the generic thing that react renders.
```ts
    type ReactNode =
        | ReactElement
        | string
        | number
        | Iterable<ReactNode>
        | ReactPortal
        | boolean
        | null
        | undefined
        | DO_NOT_USE_OR_YOU_WILL_BE_FIRED_EXPERIMENTAL_REACT_NODES[
            keyof DO_NOT_USE_OR_YOU_WILL_BE_FIRED_EXPERIMENTAL_REACT_NODES
        ];
```